flytekit/remote/remote.py

```diff
@@ -874,6 +874,7 @@ def fast_register_workflow(
         version: typing.Optional[str] = None,
         default_launch_plan: typing.Optional[bool] = True,
         options: typing.Optional[Options] = None,
+        copy_all: bool = False,
```

Comment:
@wild-endeavor Since this is new API for `fast_register_workflow`, should we just expose `FastPackageOptions` here?

Reply:
yup that sounds good. please do

Reply:
I have updated it and it works as expected. That being said, it is pretty rough compared to just `copy_all=True` or `--copy-all`

```python
from flytekit.tools.fast_registration import FastPackageOptions
remote.fast_register_workflow(
    my_workflow,
    fast_package_options=FastPackageOptions(ignores=[], copy_style="ALL")
)
```


```python
from flytekit.tools.fast_registration import FastPackageOptions
remote.fast_register_workflow(my_workflow, copy_all=True)
```

Reply:
We can pre-create some of these objects and expose them as constants to improve the experience.